seed/csharp-sdk/enum/plain-enums/src/SeedEnum/Core/StringEnumSerializer.cs

```diff
@@ -0,0 +1,26 @@
+using System.Runtime.Serialization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace SeedEnum.Core;
+
+internal class StringEnumSerializer<T> : JsonConverter<T>
```

Comment:
Looks like this serializer is getting erroneously included here with "plain enums"